pkg/services/serviceaccounts/manager/service.go

```diff
@@ -289,3 +292,38 @@ func validAPIKeyID(apiKeyID int64) error {
 	}
 	return nil
 }
+
+// hideApiKeysTabIfNoAPIKeysPresent is used to remove the apikeys tab from the admin tabs
+// if there are no apikeys, this makes it so that we only issue service account tokens instead
+func hideApiKeysTabIfNoAPIKeysPresent(l *log.ConcreteLogger, store *sqlstore.SQLStore, kvStore kvstore.KVStore) {
+	var err error
+	ctx := context.Background()
+	sess := store.GetSqlxSession()
+	orgs := make([]*org.OrgDTO, 0)
+	err = sess.Select(ctx, &orgs, "SELECT id, name FROM org")
+	if err != nil {
+		l.Error(fmt.Sprintf("unable to query for org for hiding api keys: %e", err))
+		return
+	}
+	if len(orgs) != 1 {
```

Comment:
Why don't we check for all orgs?

Reply:
i'd figured that we want to be defensive and only apply this change for the instances where grafana has never been started

Reply:
do you think, we should do it for all orgs for any instance where do we not have any apikeys and service account tokens? 🤔

Reply:
Ok, that makes sense. We only need to be carefully detecting new instances. I think it's not 100% accurate that new instance == one org. Let's try to figure this out.

Reply:
during our discussion today, we decided that we should only consider one org. So no change here